templates/openstack/support.html

```diff
@@ -0,0 +1,530 @@
+{% extends "openstack/_base_openstack.html" %}
+
+{% block title %}OpenStack support on Ubuntu{% endblock %}
+
+{% block meta_description %}Canonical is a global leader in OpenStack support, offering enterprise full-stack support subscription at the best server/annum cost.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1mHaZQ_VZnISTUEpYKiVK-Ejr5ne8lVevpxEc0LpXYKI/edit#heading=h.wtb1j7wwfobn{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-bottomed">
+  <div class="u-fixed-width">
+    <h1>OpenStack support</h1>
+    
+    <p class="p-heading--4">Use OpenStack with confidence with Ubuntu Advantage for Infrastructure</p>
+    
+    <p>One package, multiple benefits. The most comprehensive Linux enterprise subscription, covering all aspects of open infrastructure.</p>
+    
+    <p>Subscribe to Ubuntu Advantage for Infrastructure and benefit from extended security, certified compliance and 24x7 support.</p>
+
+    <a href="/openstack/support#get-in-touch" class="p-button--positive js-invoke-modal">Get OpenStack support</a>
+
+    <a href="/engage/redhat-openstack-comparison-whitepaper">Read a Red Hat OpenStack comparison whitepaper&nbsp;&rsaquo;</a>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <p class="p-muted-heading u-align--center">With enterprise support from Canonical</p>
+    <ul class="p-inline-images">
+      <li class="p-inline-images__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/7538babd-Bloomberg_logo.svg",
+          alt="Bloomberg",
+          width="130",
+          height="26",
+          hi_def=True,
+          attrs={"class": "p-inline-images__logo"},
+          loading="auto"
```

Comment:
```suggestion
          loading="lazy"
```
images that aren't at the top of the page should be set to loading='lazy'

Reply:
These can be seen on large screens, so I've switched them all to auto.